backend/src/utils/githubUtil.ts

```diff
@@ -127,6 +120,10 @@ const getNonReviewComments = async (pull: PullRequest): Promise<Comment[]> => {
     });
 };
 
+/** ="`time` is between the date range `start` to `end`." */
+export const isWithinDates = (time: number, start: number, end: number): boolean =>
+  start <= time && time <= end;
+
```

Comment:
I think this can go in a more general util file, this isn't really that specific to GitHub

Reply:
Looking in `memberUtil.ts`, the only function there that is technically not specifically related to members is `filterImagesResponse()`. Should we make a `util.ts` file for these two?

Reply:
sure sure, a general `util.ts` file will always be useful